settings.gradle

```diff
@@ -10,12 +10,7 @@ pluginManagement {
 }
 
 rootProject.name = 'sptribs-case-api'
-
-// Uncomment if you wish to debug configGenerator locally
-/*var configGenerator = '../dtsse-ccd-config-generator'
-if (file(configGenerator).exists()) {
-  includeBuild configGenerator
-}*/
+//includeBuild '../dtsse-ccd-config-generator'
```

Comment:
Is the comment you've deleted no longer relevant?  If so, should the commented-out line also be deleted?

Reply:
> Is the comment you've deleted no longer relevant? If so, should the commented-out line also be deleted?

I think it was added by Tamir, when he was experimenting locally, but will yeah I will it there in case